pkgs/tools/networking/mu/default.nix

```diff
@@ -28,8 +28,12 @@ stdenv.mkDerivation rec {
 
   # Install mug and msg2pdf
   postInstall = ''
-    cp -v toys/msg2pdf/msg2pdf $out/bin/
-    cp -v toys/mug/mug $out/bin/
+    if [ -f toys/msg2pdf/msg2pdf ]; then
+      cp -v toys/msg2pdf/msg2pdf $out/bin/
```

Comment:
I usually prefer to avoid the expected-to-fail option, but if `|| true` is a nixpkgs idiom, then we can do that. The larger point, though, is that this was previously guarded behind a nix conditional that was removed. I hope this patch is a temporary fix until equivalent logic can be replaced.